server/app/views/addTask.scala.html

```diff
@@ -14,3 +15,17 @@
         <input type="submit" value="@msg("addTask")"/>
     }
 }
+@enhanceReport(report: Option[String]) = @{
```

Comment:
we have similar code on Scala.js for report. Do you think we can move this report enhancement function to shared module to keep in one place?